Neos.ContentRepository.Core/Classes/Projection/ContentGraph/NodeTags.php

```diff
@@ -33,8 +33,8 @@
 final readonly class NodeTags implements \IteratorAggregate, \Countable, \JsonSerializable
 {
     private function __construct(
-        private SubtreeTags $tags,
-        private SubtreeTags $inheritedTags,
+        public SubtreeTags $tags,
+        public SubtreeTags $inheritedTags,
```

Comment:
How is that related? I deliberately left those private to simplify the API

Reply:
I need to check the complete set of tags against a list, see below

Reply:
I don't think that this is required, since there are `withoutInherited()` and `onlyInherited()` convenience methods.
It's not a big difference, but my goal was to make the API of `SubtreeTags` and `NodeTags` as similar as possible because in most cases you are only interested whether a node has a certain tag or not and iteration etc should always be the same.

Reply:
Ah, you're right, without/onlyInherited()->toStringArray does the job just as good. I reverted this.